daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackage.scala

```diff
@@ -222,6 +222,7 @@ private[daml] object StablePackage {
           v1_6,
         ) {
       val Tuple2: Ref.TypeConName = assertIdentifier("Tuple2")
+      val Either: Ref.Identifier = assertIdentifier("Either")
```

Comment:
That is a very slippery slope my friend, since most of stable packages is data types, you could argue we should cover every single data type here.
IMO this should just be "add as you need it"

Reply:
Note - this isn't really about the assertion, assertIdentifier isn't checking the definition exists, its just checking the name we give is valid. This is for easy access to the identifiers